app/assets/javascripts/all.js

```diff
@@ -5,7 +5,7 @@
 // the compiled file.
 //
 //= require govuk-admin-template
-//= require vendor/jquery/jquery-ui-1.10.2.custom
+//= require jquery.ui.all
```

Comment:
The commit message for this change mentions a gem. Would that be `jquery-ui-rails`? Out of interest what prompted this change?

Reply:
Yeah, `jquery-ui-rails` was upgraded to work with Rails 5 and I also took the opportunity to switch to using the version of jQuery UI that the gem provides rather than a static version built into whitehall